### PR TITLE
Show shared collections under favourites. Design tweaks

### PR DESCRIPTION
**Pull request by eronisko**
image
image
localhost_8000_oblubene_ids%5B%5D=SVK%3ASNG O_1873 ids%5B%5D=SVK%3ASNG O_2174 ids%5B%5D=SVK%3ASNG O_1877 ids%5B%5D=SVK%3ASNG O_1881 ids%5B%5D=SVK%3ASNG O_2169 ids%5B%5D=SVK%3ASNG O_2168 ids%5B%5D=SVK%3ASNG O (1)

### CODE REVIEW

File: resources/js/components/user-collections/SharedCollection.vue [hunk 1]
```diff
@@ -33,6 +68,11 @@ export default {
         editUrl() {
             return this.getUrlFromTemplate(this.editUrlTemplate);
         },
+        itemsCountWord() {
+            if (this.itemsCount === 1) return 'dielo'
+            if (this.itemsCount < 5) return 'diela'
+            return 'diel'
+        },
```
Comment (igor-kamil) [inline resources/js/components/user-collections/SharedCollection.vue:75]: this is nice. however duplication (necessary to maintain in two places). 
what do you think about covering this using `trans_choice()` in backend? 
(as far as I know `laravel-vue-lang` doesn't support it)
just a question to consider in the future i18n task

File: resources/less/_colors.less [hunk 2]
```diff
@@ -1,6 +1,7 @@
 @primary: #484224;
 @dark: #333;
 @light: #fff;
+@white: #fff;
```
Comment (igor-kamil) [inline resources/less/_colors.less:4]: why not to reuse `@light` ?

File: resources/less/custom.less [hunk 3]
```diff
@@ -58,6 +58,14 @@ html {
...
 }
 
+.flex {
+  display: flex;
+}
+
+.flex-grow {
+  flex-grow: 1;
+}
+
```
Comment (igor-kamil) [inline resources/less/custom.less:68]: so much looking forward to upgrade our CSS framework....

File: resources/less/custom.less [hunk 4]
```diff
@@ -66,20 +74,31 @@ html {
...
 .color { color: @primary-color; }
+
 // Background Colors
 .bg-light {background-color: @light !important;}
 .bg-dark {background-color: @dark !important;}
 .no-border {border: none !important;}
+.bg-gray-300 {
+  background-color: @light-grey;
+}
+.bg-white {
```
Comment (igor-kamil) [inline resources/less/custom.less:91]: ok.... now I see.... getting ready for tailwind 🚀

File: resources/lang/en/user-collection.php [hunk 5]
```diff
@@ -3,11 +3,9 @@
...
-    'content-intro'    => 'Here you will find the artworks you have saved as favourites. <br>
-The artworks are only temporarily stored in your browser.',
-    'content-usage'    => 'Click on the star to remove the work from the list. <br>
-Deleted works will disappear when the page is reloaded.',
-    'content-feedback' => 'You can send bugs, comments and suggestions to <a href="mailto:lab@sng.sk"> lab@sng.sk </a>',
+    'content-intro'    => 'Your favourite artworks are only stored in this browser.',
+    'content-usage'    => 'Click on the star to remove the work from the list. Removed artworks will disappear once you reload the page.',
+    'share-info'       => 'You can create share your favourites as a selection.',
     'clear'            => 'delete all artworks',
     'clear-confirm'    => 'Are you sure you want to delete all your favourite artworks from the list?',
```
Comment (igor-kamil) [inline resources/lang/en/user-collection.php:10]: some (already used) translation strings are missing:
Screenshot 2021-11-11 at 13 28 29
(merge from `sk/user-collection.php` keys + same for `sk/general.php` ... for the next (i18n) task)

**Review by igor-kamil — APPROVED** [review]
this is great! 
just some minor i18n related issues... to be solved separately.

File: resources/js/components/user-collections/SharedCollection.vue (hunk 1 above)
Comment (eronisko) [reply resources/js/components/user-collections/SharedCollection.vue:75]: you're totally right. I'll tackle this in the i18n effort

Comment (eronisko) [reply resources/js/components/user-collections/SharedCollection.vue:75]: you're totally right. I'll tackle this in the i18n effort

File: resources/less/custom.less (hunk 4 above)
Comment (eronisko) [reply resources/less/custom.less:91]: yes :)

**Pull request merged**